CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ Note that this project **does not** adhere to [Semantic Versioning](https://semv
 - We fixed an issue where library shown as unsaved and marked (*) after accepting changes made externally to the file. [#11027](https://github.com/JabRef/jabref/issues/11027)
 - We fixed an issue where drag and dropping entries from one library to another was not always working. [#11254](https://github.com/JabRef/jabref/issues/11254)
 - We fixed an issue where drag and dropping entries created a shallow copy. [#11160](https://github.com/JabRef/jabref/issues/11160)
+- We fixed an issue where imports to a custom group would only worl for the first entry [#11085](https://github.com/JabRef/jabref/issues/11085)
```

Comment:
Is one changelog entry enough? There are two issues targeted by this PR and a user reported that it fixed a third issue.